pom.xml

```diff
@@ -95,11 +94,12 @@
     <version.morphia.core>1.5.8</version.morphia.core>
     <version.xsom>20140925</version.xsom>
     <version.commonscompress>1.11</version.commonscompress>
-    <version.spring.security>5.1.1.RELEASE</version.spring.security>
-    <version.spring>5.1.2.RELEASE</version.spring>
-    <version.spring.context>5.1.2.RELEASE</version.spring.context>
-    <version.spring.beans>5.1.2.RELEASE</version.spring.beans>
-    <version.spring.test>5.1.2.RELEASE</version.spring.test>
+    <version.spring.security>5.3.2.RELEASE</version.spring.security>
+    <version.spring>5.2.6.RELEASE</version.spring>
+    <version.spring.test>${version.spring}</version.spring.test>
+    <version.spring.context>${version.spring}</version.spring.context>
+    <version.spring.beans>${version.spring}</version.spring.beans>
+    <version.spring.test>${version.spring}</version.spring.test>
```

Comment:
I think you have the same property twice?

Reply:
Fixed!